src/bio.c

```diff
@@ -108,6 +111,12 @@ typedef union bio_job {
         lazy_free_fn *free_fn; /* Function that will free the provided arguments */
         void *free_args[];     /* List of arguments to be passed to the free function */
     } free_args;
+
+    struct {
+        int type;
+        connection *conn; /* Connection to download the RDB from */
+        int dest_fd;      /* fd to save the RDB to */
```

Comment:
nit - just to avoid unnecessary alignment paddings
```suggestion
        int type;
        int dest_fd;      /* fd to save the RDB to */
        connection *conn; /* Connection to download the RDB from */
```